test/test_fat.rb

```diff
@@ -366,4 +366,38 @@ def test_change_rpath
       delete_if_exists(actual)
     end
   end
+
+  def test_add_rpath
+    pass
+  end
+
+  def test_delete_rpath
+    groups = FAT_ARCH_PAIRS.map do |arch|
+      ["hello.bin", "hello_actual.bin"].map do |fn|
+        fixture(arch, fn)
+      end
+    end
+
+    groups.each do |filename, actual|
+      file = MachO::FatFile.new(filename)
+
+      refute_empty file.rpaths
+      orig_npaths = file.rpaths.size
+
+      file.delete_rpath(file.rpaths.first)
+      assert file.rpaths.size < orig_npaths
```

Comment:
Not annoying at all!

Tell me if I missed anything after this push. I'm planning to do a lot of clean up of the test suite this coming week, so I'll apply these tips to the non-rpath tests then.